chpl/chpl-resources/src/main/resources/environment.properties.template

```diff
@@ -70,11 +70,6 @@ inheritanceReportEmailWeeklyFileName=inheritance-validation-weekly-report.csv
 inheritanceReportEmailWeeklyNoContent=<p>No listings were found that do not meet ICS requirements.</p>
 inheritanceReportEmailAcbWeeklyHtmlMessage=<h3>ONC-ACB Weekly ICS Validation Report</h3>
 
-#cache status maximum age email properties
```

Comment:
Should make a note to remove these from deployment jobs during push, but not before. Probably in the wiki?